3rdparty/mshadow/cmake/mshadow.cmake

```diff
@@ -1,34 +1,40 @@
 set(mshadow_LINKER_LIBS "")
 
-set(BLAS "Open" CACHE STRING "Selected BLAS library")
-set_property(CACHE BLAS PROPERTY STRINGS "Atlas;Open;MKL")
+set(BLAS "openblas" CACHE STRING "Selected BLAS library")
```

Comment:
This breaks backwards compatibility

Reply:
It makes it consistent with Makefile, that's the whole point of the PR. Please reconsider your veto.

Reply:
None the less, it's not Backwards compatible since it changes the input arguments, right?

Reply:
Yes, you are right. The impact should be very small.